### ISSUE TITLE
build fails due to openSUSE 42.1 repo being EOL and gone

### ISSUE THREAD

**Issue by girtsf** [RedpointGames/phabricator#82]
I noticed that my custom startup script that installed "patch" package with zypper started failing. Looks like build for the phabricator image is not working anymore either due to openSUSE 42.1 repo being gone:

```
$ docker build .
[..]
Building repository 'nodejs' cache [....done]
File '/repodata/repomd.xml' not found on medium 'http://download.opensuse.org/repositories/openSUSE:/Leap:/42.1/standard/'

Abort, retry, ignore? [a/r/i/? shows all options] (a): a
ABORT request: Aborting requested by user
```

https://en.opensuse.org/Lifetime says that 42.1 was EOLed on May 17th 2017.

**Comment by hach-que** [comment]
Bleh, I saw this happen the other day on one of my other Docker images.

Presumably it can be fixed by just changing the repos to point at 42.2 instead, though ideally I'd change the image to be Ubuntu if I had time (which I don't).

**Comment by cguentherTUChemnitz** [comment]
I would suggest to use alpine linux instead. It is a very small and secure linux distribution especially designed for docker usage. This is also the reason why several official containers are currently moving to alpine, like mariadb e.g.

best,
Christian
